[Security] Add methods param doc for isCsrfTokenValid attribute (#20811)
symfony/symfony-docs: security/csrf.rst

        // ... do something, like deleting an object
    }

> You add ``methods`` to the attribute to specify the HTTP methods that are allowed for

Reviewer: ```suggestion
You can use the ``methods`` parameter to specify the HTTP methods that are allowed for
```

> the toekn validation, :class:`Symfony\\Component\\Security\\Http\\Attribute\\IsCsrfTokenValid` is ignored for other methods. By default, the attribute allows all methods.:

Reviewer: ```suggestion
the token validation, :class:`Symfony\\Component\\Security\\Http\\Attribute\\IsCsrfTokenValid` is ignored for other methods. By default, the attribute allows all methods.:
```

> the token validation, :class:`Symfony\\Component\\Security\\Http\\Attribute\\IsCsrfTokenValid` is ignored for other methods. By default, the attribute allows all methods.:

Reviewer 2: ```suggestion
the token validation, :class:`Symfony\\Component\\Security\\Http\\Attribute\\IsCsrfTokenValid` is ignored for other methods. By default, the attribute allows all methods::
```

    #[IsCsrfTokenValid('delete-item', tokenKey: 'token', methods: ['DELETE'])]
    public function delete(Post $post): Response
    {

>         // ... do something, like deleting an object

Reviewer: ```suggestion
        // ... delete the object
```
